dark/reads.py

```diff
@@ -1512,6 +1512,30 @@ def variableSites(self, confirm=False, homogeneityLevel=1.0,
 
         return varSites
 
+    def combineReads(self):
+        """
+        Combine all reads into a single read.
+        """
+        reads = list(self)
+        length = len(reads[0].sequence)
+        sequence = ''
+        for site in range(length):
+            bases = set([r.sequence[site] for r in reads])
+            if len(bases) == 1:
+                sequence += list(bases)[0]
+            elif (len(bases) == 2 and 'N' in bases and bool(
+                  bases.intersection({'A', 'T', 'G', 'C'}))):
+                sequence += list(bases.intersection({'A', 'T', 'G', 'C'}))[0]
+            else:
+                nucleotides = set()
+                for base in bases:
+                    nucleotides.update(AMBIGUOUS.get(base, ''))
+                for ambiguity, b in AMBIGUOUS.items():
+                    if b == nucleotides:
+                        sequence += ambiguity
```

Comment:
It would be clearer what's going on if you used `set()` instead of `''` as the second arg to the `get` (but good use of `get` :-)).

It also looks to me like you could be using `BASES_TO_AMBIGUOUS` from `dark/dna.py`. As in something like:

```python
try:
    sequence += BASES_TO_AMBIGUOUS[''.join(sorted(nucleotides))]
except KeyError:
    raise ValueError('Unknown DNA base(s): %r' % nucleotides - set('ACGTN-'))
```

Your code should have had a `break` and it also doesn't check for an unknown base (AFAICT).  You should add a test for the latter and make sure my code (if you use it) raises `ValueError`.